media/css/base/mozid-modal.less

```diff
@@ -31,7 +31,8 @@ html.noscroll body {
             &> header {
                 display: block;
                 height: auto;
-                .font-size(60px);
+                .font-size(48px);
+                padding: 0 (@gridGutterWidth * 3);
```

Comment:
One addition - would be nice to drop the line height down to 1.3 or so. 1.5 is a bit much when the text wraps.